tests/integration/test_refrig_case_osw.py

```diff
@@ -95,7 +96,7 @@ def test_simple_external_clock(self):
         sleep(65)
```

Comment:
```suggestion
        sleep(30)
```

Can we reduce this to make tests run a bit faster?

Reply:
I'm fine with this. The 65 figure has been around for a while so I wasn't sure why that number, haha.